src/bin/deploy.rs

```diff
@@ -15,28 +15,32 @@ async fn main() -> Result<()> {
 
 	let create = CreateCommandOption::new(CommandOptionType::SubCommand, "create", "Creates a new blog");
 	let delete = CreateCommandOption::new(CommandOptionType::SubCommand, "delete", "Deletes your blog");
-	let nick =
-		CreateCommandOption::new(CommandOptionType::SubCommand, "nick", "Changes name of your blog").add_sub_option(
-			CreateCommandOption::new(CommandOptionType::String, "name", "New name of your blog"),
-		);
-	let webhook = CreateCommandOption::new(CommandOptionType::SubCommand, "webhook", "Get webhook for your blog");
+	let webhook = CreateCommandOption::new(CommandOptionType::SubCommand, "webhook", "Gets a webhook for your blog");
```

Comment:
```suggestion
	let webhook = CreateCommandOption::new(CommandOptionType::SubCommand, "webhook", "Gets the webhook for your blog");
```

Using `a` may imply that we are generating multiple webhooks but this is not the case